Dockerfile

```diff
@@ -46,10 +46,13 @@ COPY --from=hermes-builder /usr/bin/hermes /usr/local/bin/
 COPY --from=cometmock-builder /usr/local/bin/cometmock /usr/local/bin/cometmock
 COPY --from=gorelayer-builder /bin/rly /usr/local/bin/
 
-COPY --from=is-builder /go/bin/interchain-security-pd /usr/local/bin/interchain-security-pd
-COPY --from=is-builder /go/bin/interchain-security-cd /usr/local/bin/interchain-security-cd
-COPY --from=is-builder /go/bin/interchain-security-cdd /usr/local/bin/interchain-security-cdd
-COPY --from=is-builder /go/bin/interchain-security-sd /usr/local/bin/interchain-security-sd
+COPY --from=is-builder /go/bin/providerd /usr/local/bin/providerd
+COPY --from=is-builder /go/bin/consumerd /usr/local/bin/consumerd
+COPY --from=is-builder /go/bin/democracyd /usr/local/bin/democracyd
+COPY --from=is-builder /go/bin/sovereignd /usr/local/bin/sovereignd
```

Comment:
Could you also change sovereign to standalone? Calling a non-ICS chain sovereign, entails that consumer chains don't have sovereignty, which is not true.

Reply:
yeah I'm good with this change sir